src/Route/RoutePageGenerator.php

```diff
@@ -77,7 +77,7 @@ public function update(SiteInterface $site, ?OutputInterface $output = null, boo
             if (
                 !$this->decoratorStrategy->isRouteNameDecorable($name)
                 || !$this->decoratorStrategy->isRouteUriDecorable($route->getPath())
-                || (null !== $routeHostRegex
+                || (\is_string($routeHostRegex)
```

Comment:
null !== $routeHostRegex
&& '' !== $routeHostRegex
&& ...

Will fix the psalm error